_posts/databases/2000-01-01-mongodb-addon.md

```diff
@@ -0,0 +1,199 @@
+
+### From the command-line interface
+
+To upgrade or downgrade your addon the sub-command is the same: `addons-upgrade`.
+
+{% highlight bash %}
+$ scalingo -a example-app addons-upgrade example-app-3030 2g
+{% endhighlight %}
+
+In this example, `example-app-3030` is the ID of the addon, and `2g` is the plan we want to upgrade to.
+
+To find out the addon ID:
+
+{% highlight bash %}
+$ scalingo -a example-app addons 
+
++------------------+------------------+------+
+|       ADDON      |        ID        | PLAN |
++------------------+------------------+------+
+| Scalingo MongoDB | example-app-3030 |   1g |
++------------------+------------------+------+
+{% endhighlight %}
+
+## Database dashboard
+
+The Scalingo MongoDB dashboard is the central place for administrative tasks such as:
+
+- Monitor database and system stats
+- Upgrade the database engine version
+- Activate database specific features (e.g. Oplog for MongoDB)
+- Manage database users
+- Manage backups
+
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_database_mongo_overview.png" %}
+{% include mdl_img.html %}
+
+
+### Container Stats
+
+<table class="mdl-data-table ">
+  <tbody>
+    <tr>
+      <td class="mdl-data-table__cell--non-numeric">CPU usage</td>
+      <td class="mdl-data-table__cell--non-numeric">Current CPU usage.</td>
+    </tr>
+    <tr>
+      <td class="mdl-data-table__cell--non-numeric">Memory usage</td>
+      <td class="mdl-data-table__cell--non-numeric">Display the current, hightest and free memory. Highest is the maximum memory recorded since database restarted.</td>
+    </tr>
+    <tr>
+      <td class="mdl-data-table__cell--non-numeric">Swap usage</td>
+      <td class="mdl-data-table__cell--non-numeric">Display the current, hightest and free swap. Highest is the maximum swap recorded since database restarted.</td>
+    </tr>
+  </tbody>
+</table>
+
+### Database Stats
+
+<table class="mdl-data-table ">
+  <tbody>
+    <tr>
+      <td class="mdl-data-table__cell--non-numeric">MongoDB index size</td>
+      <td class="mdl-data-table__cell--non-numeric">Index size reported by MongoDB, for the best performance this value should fit in the memory size.</td>
+    </tr>
+    <tr>
+      <td class="mdl-data-table__cell--non-numeric">Data size</td>
+      <td class="mdl-data-table__cell--non-numeric">Logical space reported by the database.</td>
+    </tr>
+    <tr>
+      <td class="mdl-data-table__cell--non-numeric">Database on disk size</td>
+      <td class="mdl-data-table__cell--non-numeric">Effective physical space used.</td>
+    </tr>
+  </tbody>
+</table>
+
+## Activate MongoDB Oplog
+
+Oplog is a MongoDB feature which logs all the operations achieved on a MongoDB cluster. Meteor uses this feature to sync different instances of an application. If you wish to use Oplog in your non-Meteor application please refer to the official [MongoDB documentation](https://docs.mongodb.org/manual/core/replica-set-oplog/).
```

Comment:
Not clear enough:
1st : general case
2nd : specific cases (MeteorJS -- with link to Meteor page)